test/Dialect/Torch/refine-types.mlir

```diff
@@ -1004,3 +1004,16 @@ func @aten_matmul_broadcast_vector(%arg0: !torch.vtensor<[?,?,?,?,?],f32>, %arg1
   %0 = torch.aten.matmul %arg0, %arg1 : !torch.vtensor<[?,?,?,?,?],f32>, !torch.vtensor<[?],f32> -> !torch.tensor
   return %0 : !torch.tensor
 }
+
+// ----
+// CHECK-LABEL:  func @aten_prim_numtotensor(
+// CHECK-SAME:   %[[SELF:.*]]: !torch.int)
+// CHECK-SAME:   -> !torch.tensor {
+// CHECK:    %[[NTT:.*]] = torch.prim.NumToTensor.Scalar %[[SELF]] : !torch.int -> !torch.tensor<[],si64>
+// CHECK:    %[[CAST:.*]] = torch.tensor_static_info_cast %[[NTT]] : !torch.tensor<[],si64> to !torch.tensor
+// CHECK:    return %[[CAST]] : !torch.tensor
+
+func @aten_prim_numtotensor(%arg0: !torch.int) -> !torch.tensor {
```

Comment:
The convention here is to name the function after the op, so `@torch.prim.NumToTensor.Scalar`. (note: the test `matmul` before this one breaks the convention)

Reply:
@gpetters94 Could you address Ramiro's comment here? It would be great if you could fix the above tests as well. The alignments of CHECK are also off starting from `aten_matmul_broadcast_matrix` on line 982. could you help to fix that as well?

Reply:
Sure. What do you mean by CHECK being misaligned? It looks fine to me.

Reply:
I meant the space between CHECK tag and the texts.

Reply:
How does the formatting look in the most recent commit?

Reply:
looks better! Thanks!